projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -768,7 +768,7 @@ public static function maybe_show_red_bubble() {
 			self::get_red_bubble_alerts(),
 			function ( $alert ) {
 				// We don't want to show silent alerts
-				return ! $alert['is_silent'];
+				return isset( $alert['is_silent'] ) && ! $alert['is_silent'];
```

Comment:
I note this is a slight behavior change: previously this callback would return true if `$alert['is_silent']` is not set, now it will return false. Is that intended?

If that's not intended, you could simplify this:
```suggestion
				return empty( $alert['is_silent'] ) ;
```